lib/gollum/templates/page.mustache

```diff
@@ -6,93 +6,15 @@ Mousetrap.bind(['e'], function( e ) {
 });
 </script>
 <div id="wiki-wrapper" class="page">
-<div id="head"> 
-  <h1>{{page_header}}</h1>
-  <ul class="actions">
-		<li class="minibutton"><a href="{{base_url}}/"
-       class="action-home-page">Home</a></li>
-    <li class="minibutton">
-      {{>searchbar}}
-    </li>
-    <li class="minibutton"><a href="{{pages_path}}"
-      class="action-all-pages">Overview</a></li>
-    {{#allow_editing}}
-      <li class="minibutton jaws">
-        <a href="#" id="minibutton-new-page">New</a></li>
-    {{/allow_editing}}
-    {{#allow_editing}}
-      {{#allow_uploads}}
-      <li class="minibutton jaws">
-        <a href="#" id="minibutton-upload-page">Upload</a></li>
-      {{/allow_uploads}}
-    {{/allow_editing}}
-    {{#allow_editing}}
-      {{#editable}}
-        <li class="minibutton jaws">
-          <a href="#" id="minibutton-rename-page">Rename</a></li>
-      {{/editable}}
-    {{/allow_editing}}
-    {{#allow_editing}}
-      {{#editable}}
-        <li class="minibutton"><a href="{{edit_path}}/{{escaped_url_path}}"
-           class="action-edit-page">Edit</a></li>
-      {{/editable}}
-    {{/allow_editing}}
-    <li class="minibutton"><a href="{{history_path}}/{{escaped_url_path}}"
-       class="action-page-history">History</a></li>
-    <li class="minibutton"><a href="{{latest_changes_path}}"
-       class="action-page-history">Latest Changes</a></li>
-  </ul>
-  <div class="breadcrumb">{{{breadcrumb}}}</div>
+<div id="head">
+	{{#navbar?}}{{>navbar}}{{/navbar?}}
 </div>
-<div id="wiki-content">
-<div class="{{#has_header}}has-header{{/has_header}}{{#has_footer}} has-footer{{/has_footer}}{{#has_sidebar}} has-sidebar has-{{bar_side}}bar{{/has_sidebar}}{{#has_toc}} has-toc{{/has_toc}}">
-  {{#has_toc}}
-  <div id="wiki-toc-main">
-    {{{toc_content}}}
-  </div>
-  {{/has_toc}}
-  {{#has_sidebar}}
-  <div id="wiki-sidebar" class="gollum-{{sidebar_format}}-content">
-    <div id="sidebar-content" class="markdown-body" dir="auto">
-      {{{sidebar_content}}}
-    </div>
-  </div>
-  {{/has_sidebar}}
-  <div id="wiki-body" class="gollum-{{format}}-content">
-    {{#has_header}}
-    <div id="wiki-header" class="gollum-{{header_format}}-content">
-      <div id="header-content" class="markdown-body" dir="auto">
-        {{{header_content}}}
-      </div>
-    </div>
-    {{/has_header}}
-    <div class="markdown-body{{#header_enum?}} header-enum{{/header_enum?}}" {{#header_enum?}}style="--header-enum-style:{{header_enum_style}};"{{/header_enum?}} dir="auto">
-      {{{rendered_metadata}}}
-      {{{content}}}
-    </div>
-  </div>
-  {{#has_footer}}
-  <div id="wiki-footer" class="gollum-{{footer_format}}-content">
-    <div id="footer-content" class="markdown-body" dir="auto">
-      {{{footer_content}}}
-    </div>
-  </div>
-  {{/has_footer}}
-  </div>
 
-</div>
-<div id="footer">
```

Comment:
We could
1. wrap the footer in `{{^preview}}{{/preview}}`
2. add a `preview` method to the Preview view (`preview.rb`) that always returns true.
to avoid rendering the footer on preview pages.
Could also implement that solution in a new PR

Reply:
There is no `preview.rb` view, only `page`. So we would have to differentiate within the `page.rb` view, but I suppose we could just check if the Page object is a PreviewPage, correct?

Reply:
Ah, good point. Or just add an `attr_reader :preview` to the Page view, and only set it to `true` in the `/preview` route?

Reply:
I'll see what I can do!

Reply:
> just add an attr_reader :preview to the Page view, and only set it to true in the /preview route

Works like a charm.